@uportal/content-carousel/src/components/ContentCarousel.scss

```diff
@@ -22,15 +22,29 @@
     z-index: 2;
   }
 }
-
-.slick-item {
-  padding: 10px;
-
-  .slick-item-description {
-    min-height: 7rem;
+.content {
+  .slick-slide {
+    width: 300px;
+    margin: 0 10px;
+    border: 1px solid rgba(0,0,0,.125);
```

Comment:
:art: spacing between items in list `rgba(0, 0, 0, .125)`
:art: zero prefix fractional values `0.125`